assets/airlines/fji.json

```
@@ -0,0 +1,34 @@
+{
+    "icao": "fji",
+    "name": "Fiji Airways",
+    "callsign": {
+        "name": "Fiji",
+        "callsignFormats": [
+            "###",
+            "####"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["A332", 5],
+            ["A333", 1],
+            ["A359", 2],
+            ["B38M", 2],
+            ["B737", 1],
+            ["B738", 2]
+        ],
+        "short": [
+            ["B38M", 2],
+            ["B737", 1],
+            ["B738", 2]
+        ],
+        "long": [
+            ["A332", 5],
+            ["A333", 1],
+            ["A359", 2]
+        ],
+        "lax": [
```

```suggestion
        "a359": [
```
Since this fleet only contains the A359, it might be better to name it "a359". That way the fleet can be used in other airports as well and I think it's always good to try to keep the number of fleets low (e.g. faster airline updating)